src/distance.hpp

```diff
@@ -66,7 +84,7 @@ class Distance {
      * @param[in]     Distance::Scale       The scale in which the distance has to be returned
      * @return        inches
      */
-    int convertToInches(int centimeters);
+    static int convertToInches(int centimeters);
```

Comment:
Doxygen mentions two parameters, but function only takes one. It seems like the scale parameter is unnecessary.